packages/cardano-services/test/cli.test.ts

```diff
@@ -2640,6 +2640,65 @@ describe('CLI', () => {
         done();
       });
     });
+    it('exits with code 1 without METADATA_FETCH_MODE', (done) => {
```

Comment:
Does it need to be explicit? We can just default to direct

Reply:
It does not need to be explicit. It defaults to DIRECT (mentioned you on the line) in the file packages/cardano-services/src/Program/options/stakePoolMetadata.ts

Testing just for the purpose of 'if somehow it was missing'.

Reply:
> Testing just for the purpose of 'if somehow it was missing'.

🤔 isn't the point of a default to allow the config to be optional?

Also, it's best to add `expect.assertions(2)`  at the top of the function so Jest will fail if the assertions are not executed. It protects against false-positives with async code

Reply:
> 🤔 isn't the point of a default to allow the config to be optional?
Image a case where someone mistakenly removed default params while refactoring program options. Doesn't it make sense? I can remove the check ^^ if not satisfied.

I will add `expect.assertions(2)`.

Reply:
But how does not providing it now produce a `MissingProgramOption: pool-metadata requires the medata-fetch-mode program option` error if there's a default :thinking: ?

Reply:
I thought that we were providing a manually crafted set of parameters to the CLI in the test and skipping the parameter parsing part. So ignored this scenario. But now checking deeper.